lib/interfaces/bloc.dart

```diff
@@ -0,0 +1,4 @@
+abstract class Bloc{
+  Future<void> initialize();
```

Comment:
Why you use this initialize as Future?

Reply:
Honestly i don't know because i don't implemented nothing yet, in Javi's video he treats the method as a Future, that's why.